src/internal_events/redis.rs

```diff
@@ -3,43 +3,6 @@ use vector_core::internal_event::InternalEvent;
 
 use vector_common::internal_event::{error_stage, error_type};
 
-#[derive(Debug)]
-pub struct RedisSendEventError<'a> {
-    error: &'a redis::RedisError,
-    error_code: String,
-}
```

Comment:
Duplicate from underlying sink framework?

Reply:
Correct - specifically `SinkSendError` in the retry logic.

The reason I'm adding the following comment to every sink I audit that this applies to, is to hopefully prevent future attempts to add Error emission in the service call function:

```
// Emission of Error internal event is handled upstream by the caller
```

Reply:
Just making sure, tying your comments in the PR to the changes 😄

Reply:
I probably should have mentioned that in the comment 0, next time.

Reply:
Meh, it seemed obvious - I just wanted to double check